.github/workflows/release.yml

```diff
@@ -17,13 +23,35 @@ jobs:
         run: sudo apt-get install moreutils
       - name: Set env
         run:  echo RELEASE_VERSION=${GITHUB_REF#refs/*/} >> $GITHUB_ENV && echo $RELEASE_VERSION
-      - name: "Create ZIP"
+
+      - name: Setup Python ${{ env.PYTHON_VERSION }} Environment
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: 'Resolve Project Dependencies Using Pip'
+        shell: bash
         run: |
-          (cd ${{ env.DIR_NAME }} && zip -r ../${{ env.ZIP_FILE_NAME }} .)
+          pushd './${{ env.AZURE_FUNCTIONAPP_PACKAGE_PATH }}'
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt --target=".python_packages/lib/site-packages"
+          popd
+
+
       - name: Update the WEBSITE_RUN_FROM_PACKAGE token in the arm.json file.
         env: 
           URL: ${{ github.server_url }}/${{ github.repository }}/releases/download/${{ env.RELEASE_VERSION }}/${{ env.ZIP_FILE_NAME }}
-        run: jq --arg url "$URL" '.resources[4].resources[0].properties.WEBSITE_RUN_FROM_PACKAGE=$url' ./arm_template/${{ env.JSON_FILE_NAME }} | sponge ${{ env.JSON_FILE_NAME }}
+        run: |
+          jq --arg url "$URL" '.resources[4].resources[0].properties.WEBSITE_RUN_FROM_PACKAGE=$url' ./arm_template/${{ env.JSON_FILE_NAME }} | sponge ${{ env.JSON_FILE_NAME }}
+          jq --arg version "$RELEASE_VERSION" '.contentVersion=$version' ./arm_template/${{ env.JSON_FILE_NAME }} | sponge ${{ env.JSON_FILE_NAME }}
```

Comment:
this will reset the WEBSITE_RUN_FROM_PACKAGE
it should be done this way:
jq --arg version "$RELEASE_VERSION" '.contentVersion=$version' ${{ env.JSON_FILE_NAME }} | sponge ${{ env.JSON_FILE_NAME }}